src/components/Settings/GameBoardSettings.vue

```diff
@@ -27,6 +29,55 @@
       </select>
     </div>
 
+    <div class="setting">
+      <i class="fas fa-bars" />
+      <h5>Header position</h5>
+
+      <select v-model="value[platform.code].position" @change="$emit('save')">
```

Comment:
I wonder if this setting should only apply to the mobile breakpoint, the current layout works fine for desktop, however, I can definitely see the benefit of having the header down on mobile devices (within finger's reach). What do you think about having this setting only affect mobile? and maybe hide setting in desktop?

Reply:
Sure, that sounds like a good idea.